llvm/lib/Target/AMDGPU/AMDGPUCallLowering.cpp

```diff
@@ -633,7 +633,7 @@ bool AMDGPUCallLowering::lowerFormalArguments(
     const bool InReg = Arg.hasAttribute(Attribute::InReg);
 
     // SGPR arguments to functions not implemented.
-    if (!IsGraphics && InReg)
```

Comment:
This is the wrong change for this particular patch. You want the AllocateReg above determineAssignments